addons/web/static/tests/legacy/fields/relational_fields/field_one2many_tests.js

```diff
@@ -10092,6 +10092,45 @@ QUnit.module('fields', {}, function () {
             form.destroy();
             assert.verifySteps(["willUnmount 1"]);
         });
+
+        QUnit.test('o2m with sub view, with <create> tag and common default_* key in the context: ensure <create> value is the one kept', async function (assert) {
+            assert.expect(1);
+
+            var form = await createView({
+                View: FormView,
+                model: 'partner',
+                data: this.data,
+                arch: '<form>' +
```

Comment:
you can use template strings:
```js
arch: `
    <form>
        <sheet>
        ...
        </sheet>
    </form>`;
```
No need to escape the single quotes in the template then